src/commands/gegenstand.ts

```diff
@@ -60,6 +61,18 @@ export default class GegenstandCommand implements ApplicationCommand {
                         .setDescription("Die Sau, die du benutzen möchtest")
                         .setAutocomplete(true),
                 ),
+        )
+        .addSubcommand(
+            new SlashCommandSubcommandBuilder()
+                .setName("ausrüsten")
```

Comment:
Vielleicht passt hier "anlegen" oder "tragen" besser. Zumindest glaube ich, dass sich `/gegenstand anlegen` oder so verständlicher ist als `/gegenstand ausrüsten`

Reply:
Oder kollidiert das mit "benutzen"?

Reply:
anlegen finde ich tatsächlich passender. 

Meinst du das benutzen und anlegen eigenlich das selbe tut? noch könnte man beides in ein command machen, aber potentiell gibt es items die man im chat und im kampf nutzen kann, daher würd ich das getrennt lassen

Reply:
Benutzen ist etwas, was in dem moment eine Aktion ausführt. Also z.b. ein Geschenk in den Chat droppt